Sources/AblyAssetTrackingInternal/PresenceData.swift

```diff
@@ -4,14 +4,17 @@ import AblyAssetTrackingCore
 public struct PresenceData: Codable {
     public let type: ClientType
     public let resolution: Resolution?
+    public let rawLocations: Bool?
 
-    public init(type: ClientType) {
+    public init(type: ClientType, rawLocations: Bool? = nil) {
```

Comment:
name `rawLocations` implies an array of locations. I'd suggest `isRawLocations`

Reply:
This is imposed by JSON schema. Of course, I can map it on encoding but is this worth changing?